script.js

```diff
@@ -54,9 +64,91 @@ class BookStore {
 
     this.bookContentContainer.innerHTML = this.newBookDiv;
   }
+
+  getCurrentDateTime = () => {
```

Comment:
- [ ] As stated in the requirements:  the current date displayed should use the Luxon library You ought to follow this [link](https://moment.github.io/luxon/#/install) for setup.